Sources/Request+Locale.swift

```diff
@@ -0,0 +1,45 @@
+import Vapor
+import Lingo
+
+extension Request {
+    var locale: String {
+        if let session = try? session(), let sessionlocale = session["locale"] {
+            return sessionlocale
+        } else {
```

Comment:
I would suggest removing the `else` from here as we anyway return in the first `if let` block. This would make one level of indentation less for almost whole method.

Reply:
Done.